ui/components/ui/site-origin/site-origin.js

```diff
@@ -1,16 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classnames from 'classnames';
 import Chip from '../chip';
 import IconWithFallback from '../icon-with-fallback';
 
-export default function SiteOrigin({ siteOrigin, iconSrc, iconName }) {
+export default function SiteOrigin({
+  siteOrigin,
+  iconSrc,
+  iconName,
+  chip,
+  className,
+  title,
+}) {
   return (
-    <div className="site-origin">
-      <Chip
-        label={siteOrigin}
-        maxContent={false}
-        leftIcon={<IconWithFallback icon={iconSrc} name={iconName} size={32} />}
-      />
+    <div className={classnames('site-origin', className)} title={title}>
+      {chip ? (
+        <Chip
+          label={siteOrigin}
+          maxContent={false}
+          leftIcon={
+            <IconWithFallback
+              icon={iconSrc}
+              name={iconName}
+              size={32}
+              className={className}
```

Comment:
Should this be here? Additional className should usually be added to the root element?
```suggestion
```

Reply:
Yup whoops!